src/main/java/com/buddy/api/domains/profile/enums/ProfileTypeEnum.java

```diff
@@ -0,0 +1,7 @@
+package com.buddy.api.domains.profile.enums;
+
+public enum ProfileTypeEnum {
+    ADOPTER,
```

Comment:
Na PR #124 eu notei que esquecemos de modificar essa parte. Na realidade deveria se 
```suggestion
    USER,
```
Para bater com as constraints do banco. Inclusive porque paramos de distinguir entre "adopters" e "shelters" para uma distinção entre "usuário normal" e "abrigo".
Talvez para não ir errado para a `main`, valha a pena já corrigir nessa PR aqui?